tests/functional/sca/test_sca_scan_all.py

```diff
@@ -50,3 +55,21 @@ def test_scan_all_json_output(dummy_sca_repo: Repository) -> None:
     assert len(parsed_result["found_package_vulns"]) == 1
     assert parsed_result["found_package_vulns"][0]["location"] == "Pipfile.lock"
     assert len(parsed_result["found_package_vulns"][0]["package_vulns"]) == 1
+    jsonschema.validate(parsed_result, sca_scan_all_json_schema)
+    # Autoignore feature attributes are not included in the JSON output
+    assert "source_found" not in parsed_result
+    assert all(
+        [
+            key
+            not in parsed_result["found_package_vulns"][0]["package_vulns"][0]["vulns"][
+                0
+            ]
+            for key in [
+                "url",
+                "status",
+                "ignored_until",
+                "ignore_reason",
+                "ignore_comment",
+            ]
+        ]
+    )
```

Comment:
I think you can remove this code: validating the jsonschema takes care of it (but double-check just to be sure 😉)

Same below.

Reply:
Yes it does throw on additional properties